src/sentry/conf/server.py

```diff
@@ -4,19 +4,27 @@
 
 import os
 import os.path
+import random
 import re
 import socket
 import sys
 import tempfile
 from datetime import timedelta
 from urllib.parse import urlparse
 
+from celery.schedules import crontab
 from django.conf.global_settings import *  # NOQA
+from django.urls import reverse_lazy
+
+# Queue configuration
+from kombu import Exchange, Queue
```

Comment:
Can we back out the import reordering? While it likely wont have any negative effect, it's also not necessary for this patch. The only change in this file is an added comment.